Error:   18:52  error  'res' is defined but never used

> what-surv@0.0.1 lint
> eslint "{src,apps,libs,test}/**/*.ts" --fix


/home/runner/work/what-surv-backend/what-surv-backend/src/auth/auth.controller.ts
Error:   13:2[8](https://github.com/what-surv/what-surv-backend/actions/runs/7787221048/job/21233859742?pr=7#step:5:9)  error  'req' is defined but never used  @typescript-eslint/no-unused-vars
Error:   13:4[9](https://github.com/what-surv/what-surv-backend/actions/runs/7787221048/job/21233859742?pr=7#step:5:10)  error  'res' is defined but never used  @typescript-eslint/no-unused-vars
Error:   18:52  error  'res' is defined but never used  @typescript-eslint/no-unused-vars

✖ 3 problems (3 errors, 0 warnings)

Error: Process completed with exit code 1.

auth.controller.ts 내부 에러 수정하겠습니다.

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -21,5 +21,6 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
   },
 };

--- a/src/auth/auth.controller.ts
+++ b/src/auth/auth.controller.ts
@@ -5,21 +5,19 @@ import { Request, Response } from 'express';
 
 @Controller('auth')
 export class AuthController {
-    constructor(
-        private readonly authService: AuthService
-    ) {}
+  constructor(private readonly authService: AuthService) {}
 
-    // 구글 로그인
-    @Get("/login/google")
-    @UseGuards(AuthGuard('google'))
-    async loginGoogle(@Req() req: Request, @Res() res: Response) { }
+  // 구글 로그인
+  @Get('/login/google')
+  @UseGuards(AuthGuard('google'))
+  async loginGoogle(@Req() _req: Request, @Res() _res: Response) {}
 
-    // 구글 로그인 redirect
-    @Get("/callback/google")
-    @UseGuards(AuthGuard('google'))
-    async callbackGoogle(@Req() req: Request, @Res() res: Response) {
-        const user = req.user;
+  // 구글 로그인 redirect
+  @Get('/callback/google')
+  @UseGuards(AuthGuard('google'))
+  async callbackGoogle(@Req() req: Request, @Res() _res: Response) {
+    const user = req.user;
 
-        console.log(user);        
-    }
+    console.log(user);
+  }
 }

